crypto/src/hash/sha3/mod.rs

```diff
@@ -13,6 +13,7 @@ impl Sha3Hasher {
         Self
     }
 
+    #[allow(clippy::manual_div_ceil)]
```

Comment:
Similarly, in line 19 we can simplify to:
```rust
let ell = len_in_bytes.div_ceil(b_in_bytes);
```

Reply:
Same as before, it is [here](https://github.com/lambdaclass/lambdaworks/pull/944/commits/cf35df8e506cd61bf54d872de4002b99f64ce9a3).